src/models/notifications.js

```diff
@@ -0,0 +1,43 @@
+'use strict'
+
+const log = require('../services/log')('subscriptions')
+const uri = require('../services/uriManager')
+const NotFoundError = require('five-bells-shared/errors/not-found-error')
+const UnauthorizedError = require('five-bells-shared/errors/unauthorized-error')
+const Notification = require('./db/notification').Notification
+const Transfer = require('./db/transfer').Transfer
+
+function isOwnerOrAdmin (requestingUser, subscription) {
```

Comment:
This method is taken from the `subscriptions` model. It may make sense to put it in a separate file of library methods that both the `notifications` model and the `subscriptions` model can access, to avoid code duplication.